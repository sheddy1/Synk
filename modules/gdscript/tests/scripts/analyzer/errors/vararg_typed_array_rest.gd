# Currently not supported.
func f(...args: Array[int]):
	pass

func test():
	pass
