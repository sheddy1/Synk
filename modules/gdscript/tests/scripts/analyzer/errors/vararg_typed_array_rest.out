GDTEST_ANALYZER_ERROR
The rest parameter type must be "Array", but "Array[int]" is specified.
